src/components/clusterDetail/ClusterProgress.tsx

```diff
@@ -195,10 +220,19 @@ type ClusterProgressProps = {
 
 const ClusterProgress: React.FC<ClusterProgressProps> = ({ cluster, minimizedView = false }) => {
   const { status, monitoredOperators = [] } = cluster;
-  const hosts = getEnabledHosts(cluster.hosts);
-  const progressPercent = React.useMemo(() => Math.round(getProgressPercent(hosts)), [hosts]);
-  const label = getProgressLabel(cluster, progressPercent);
-  const isWorkersPresent = hosts && hosts.some((host) => host.role === 'worker');
+  const hostsProgressPercent = React.useMemo(() => Math.round(getProgressPercent(cluster.hosts)), [
+    cluster.hosts,
+  ]);
+  const operatorsProgressPercent = React.useMemo(
+    () => Math.round(getOperatorsProgressPercent(monitoredOperators)),
+    [monitoredOperators],
+  );
+  const label = getProgressLabel(
+    cluster,
+    hostsProgressPercent * 0.75 + operatorsProgressPercent * 0.25,
```

Comment:
I think it would be better to do `Math.round` only here for better results

Reply:
With current calculation we're getting percentages such as this:
<img width="378" alt="Screenshot 2021-05-13 at 12 47 04" src="https://user-images.githubusercontent.com/1121740/118115630-7d8bd500-b3e9-11eb-9f3f-268dae926496.png">